packages/strategy-tests/src/lib.rs

```diff
@@ -1072,23 +1071,41 @@ impl Strategy {
 
                     // Generate state transition for identity transfer operation
                     OperationType::IdentityTransfer if current_identities.len() > 1 => {
-                        let identities_clone = current_identities.to_owned();
-                        // Sender is the first in the list, which should be loaded_identity
-                        let owner = &mut current_identities[0];
-                        // Recipient is the second in the list
-                        let recipient = &identities_clone[1];
                         for _ in 0..count {
+                            let identities_count = current_identities.len();
+                            if identities_count == 0 {
+                                break;
+                            }
+
+                            // Select a random identity from the current_identities for the sender
+                            let random_index_sender = thread_rng().gen_range(0..identities_count);
```

Comment:
don't you want to use rng, so that tests are determinisitic based on the seed?

Reply:
Fixed. Changed to rng